receiver/otlpreceiver/otlp.go

```diff
@@ -163,6 +166,11 @@ func (r *otlpReceiver) startProtocolServers(host component.Host) error {
 // Start runs the trace receiver on the gRPC server. Currently
 // it also enables the metrics receiver too.
 func (r *otlpReceiver) Start(_ context.Context, host component.Host) error {
+	r.onceLogLocalHost.Do(func() {
```

Comment:
What is the purpose of this log? Why not log once per process if the gate is disabled?

Reply:
Now logging once when the Collector starts